Library/Homebrew/development_tools.rb

```diff
@@ -85,19 +105,23 @@ def non_apple_gcc_version(cc)
       end
     end
 
+    sig { void }
     def clear_version_cache
       @clang_version = @clang_build_version = nil
       @non_apple_gcc_version = {}
     end
 
+    sig { returns(T::Boolean) }
     def curl_handles_most_https_certificates?
       true
     end
 
+    sig { returns(T::Boolean) }
     def subversion_handles_most_https_certificates?
       true
     end
 
+    sig { returns(T::Hash[String, T.untyped]) }
```

Comment:
```suggestion
    sig { returns(T::Hash[String, T.nilable(String)]) }
```
and append `.to_s` to CPU family.